tests/test_samples.py

```diff
@@ -911,3 +936,23 @@ def test_logL_list():
 
     samples = NestedSamples(data=data, logL=logL, logL_birth=logL_birth)
     assert_array_equal(default, samples)
+    plt.close("all")
+
+
+def test_hist_plotting():
+    samples = NestedSamples(root='./tests/example_data/pc')
+    samples[['x0', 'x1', 'x2', 'x3', 'x4']].plot.hist()
+    samples.x0.plot.kde(subplots=True)
+
+    samples.plot.hist_2d('x0', 'x1')
+    samples.plot.kde_2d('x0', 'x1')
+    samples.plot.kde_1d()
+    samples.plot.hist_1d()
+
+    try:
+        samples.plot.fastkde_2d('x0', 'x1')
+        samples.plot.fastkde_1d()
+    except ImportError:
+        pass
+
+    plt.close("all")
```

Comment:
* delete this and hope for 100% coverage
* create an issue that this is not perfect pandas behaviour

Reply:
> * delete this and hope for 100% coverage

Failed. Implemented more thorough tests in [aa24d72](https://github.com/williamjameshandley/anesthetic/pull/207/commits/aa24d724a9e7f6ab005b3e441b5394325b2165d0) instead.